hedera-node/src/main/java/com/hedera/services/grpc/NettyGrpcServerManager.java

```diff
@@ -41,18 +46,21 @@
 	private Server server;
 	private Server tlsServer;
 	private final Consumer<Thread> hookAdder;
+	private final NodeLocalProperties nodeProperties;
 	private final List<BindableService> bindableServices;
 	private final ConfigDrivenNettyFactory nettyBuilder;
 	private final List<ServerServiceDefinition> serviceDefinitions;
 
 	public NettyGrpcServerManager(
 			Consumer<Thread> hookAdder,
+			NodeLocalProperties nodeProperties,
 			List<BindableService> bindableServices,
 			ConfigDrivenNettyFactory nettyBuilder,
 			List<ServerServiceDefinition> serviceDefinitions
 	) {
 		this.hookAdder = hookAdder;
 		this.nettyBuilder = nettyBuilder;
+		this.nodeProperties = nodeProperties;
```

Comment:
nit: Maybe we could save only `nettyStartRetries` and `nettyStartRetryIntervalMs` in this class. We don't need the whole `nodeProperties` object.

Reply:
Captured properties in constructor instead.